flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/ProjectionTranslator.scala

```diff
@@ -418,4 +418,30 @@ object ProjectionTranslator {
       case e: Expression => e
     }
   }
+
+
+  /**
+    * Unwrap a Call to ScalarFunctionCall, TableFunctionCall, AggFunctionCall or the built-in
+    * expression representation.
+    *
+    * @param field    the expression to unwrap
+    * @param tableEnv the TableEnvironment
+    * @return the unwrapped expression
+    */
+  def unwrap(
+      field: Expression,
+      tableEnv: TableEnvironment): Expression = {
+    field match {
+      case Alias(child, name, extraNames) =>
```

Comment:
Alias(child, name, extraNames) -> Alias(child, _, _) because name and extraNames never used.